h2o-algos/src/main/java/hex/glrm/GLRM.java

```diff
@@ -1303,7 +1426,7 @@ private Chunk chk_xnew(Chunk[] chks, int c) {
 
       // Compute new y_j values using proximal gradient
       for (int j = 0; j < _ytnew.length; j++) {
-        double[] u = new double[_ytnew[0].length];
+        double[] u = new double[_ytnew[0].length];  // Do not touch this memory allocation.  Needed for proper function.
```

Comment:
I made the change and ran test runit_pubdev_1692.R.  It always fail if I make the change.  I am leaving it alone for now and plan to come back and visit it later.